### ISSUE TITLE
Create MVP About Page

### ISSUE THREAD

**Issue by lottycoupat** [alan-turing-institute/AutSPACEs#133]
### Summary

The About page on the MVP will contain information about the project as well as a helpful resources. 

The design of the page should follow the [figma board](https://www.figma.com/file/HxqTSdeyAUNMualnBV0PET/Autspace-(WIP)?node-id=20%3A662). 

Content for the page is not yet chosen, therefore what should just be developed is a blank page, as in the figma page above, and should link with the navigation page. 

### Who can help?

Anyone interested in platform development.

**Comment by lottycoupat** [comment]
@anoura12 following our conversation this morning, I have created this issue - please add detail to it where you find necessary. 

@jhlink FYI.

**Comment by jhlink** [comment]
@lottycoupat , Is there any additional information that you want to showcase beyond what's covered in the Home page? 
Otherwise, I recommend closing this for the sake of completing milestone #2.

**Comment by GeorgiaHCA** [comment]
I agree @jhlink - instead of closing though I'm relabelling/moving this to Milestone 6 as it can be a further iteration. It's best to deploy the MVP first and we can collect more information about whether we need a separate about page and what would go in it via. user testing.

**Comment by GeorgiaHCA** [comment]
<img width="584" alt="Screenshot 2021-07-29 at 11 39 23" src="https://user-images.githubusercontent.com/46889966/127478236-f1a6191f-2774-4724-b623-9532d5f90c55.png">

**Comment by GeorgiaHCA** [comment]
Collated info is on this page: https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/529

**Comment by Chandresh726** [comment]
I would like to work on this issue
